File: specification/confluent/resource-manager/Microsoft.Confluent/preview/2021-09-01-preview/examples-js/Organization_Get.js
const { ConfluentManagementClient } = require("@azure/arm-confluent");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Get the properties of a specific Organization resource.
 *
 * @summary Get the properties of a specific Organization resource.
 * x-ms-original-file: specification/confluent/resource-manager/Microsoft.Confluent/preview/2021-09-01-preview/examples/Organization_Get.json
 */
async function organizationGet() {
  const subscriptionId = "00000000-0000-0000-0000-000000000000";
  const resourceGroupName = "myResourceGroup";
  const organizationName = "myOrganization";
  const credential = new DefaultAzureCredential();
  const client = new ConfluentManagementClient(credential, subscriptionId);
  const result = await client.organization.get(resourceGroupName, organizationName);
  console.log(result);
}

organizationGet().catch(console.error);

File: specification/confluent/resource-manager/Microsoft.Confluent/preview/2021-09-01-preview/examples-java/Organization_Get.java
import com.azure.core.util.Context;

/** Samples for Organization GetByResourceGroup. */
public final class Main {
    /*
     * x-ms-original-file: specification/confluent/resource-manager/Microsoft.Confluent/preview/2021-09-01-preview/examples/Organization_Get.json
     */
    /**
     * Sample code: Organization_Get.
     *
     * @param manager Entry point to ConfluentManager.
     */
    public static void organizationGet(com.azure.resourcemanager.confluent.ConfluentManager manager) {
        manager.organizations().getByResourceGroupWithResponse("myResourceGroup", "myOrganization", Context.NONE);
    }
}
